Src/UnitTests/CoravelUnitTests/Queuing/CancellableInvocableForQueueTests.cs

```diff
@@ -12,6 +12,25 @@ namespace UnitTests.Queuing
 {
     public class CancellableInvocableForQueueTests
     {
+        [Fact]
+        public async Task CanCancelInvocableWithPayload()
+        {
+            var services = new ServiceCollection();
+            services.AddTransient<TestCancellableInvocable>();
+            services.AddTransient<TestCancellableInvocableWithPayload>();
+            var provider = services.BuildServiceProvider();
+
+            var queue = new Queue(provider.GetRequiredService<IServiceScopeFactory>(), new DispatcherStub());
+            var payload = new TestPayload()
+            {
+                Code = "test"
+            };
+            queue.QueueInvocableWithPayload<TestCancellableInvocableWithPayload, TestPayload>(payload);
```

Comment:
Like the other existing tests, could you queue up 3 (basically copy-paste this line 2 more times) and then assert that 3 cancellations occurred? Thanks!

Reply:
Thanks, I'm already update it but i forgot move it to another commit, you can check it in [this](https://github.com/lengockyquang/coravel/commit/3664e888dff8ff26d7052b116f78a70c689cae24)